WCFDataService/Client/System/Data/Services/Client/ClientEdmModel.cs

```diff
@@ -443,7 +447,8 @@ private EdmTypeCacheValue GetOrCreateEdmTypeInternal(IEdmStructuredType edmBaseT
                             List<IEdmProperty> loadedProperties = new List<IEdmProperty>();
                             foreach (PropertyInfo property in ClientTypeUtil.GetPropertiesOnType(type, /*declaredOnly*/edmBaseType != null).OrderBy(p => p.Name))
                             {
-                                IEdmProperty edmProperty = this.CreateEdmProperty(complexType, property);
+                                FieldInfo backingField = FindBackingField(property, this.ProxyBackingFieldNamingConvention);
```

Comment:
Seems this is for complex property. Is it in scope of this change? You mentioned in one of your comments that complex property support lazy initializing already. "Auto generated proxies (from DataSvcUtil or Connected Service) does not lazy construct entity Collections (DataServiceCollection<> or ObservableCollection<>). It does however do lazy constriction of Complex types." .Does it use the same mechanism?